packages/cli/src/commands/generate.js

```diff
@@ -12,6 +12,18 @@ export default {
       default: true,
       description: 'Only generate pages for dynamic routes. Nuxt has to be built once before using this option'
     },
+    development: {
```

Comment:
To prevent confusions with `dev-development` i suggest renaming this option to `dev-tool`